src/bin/cobalt/main.rs

```diff
@@ -227,9 +308,9 @@ fn run() -> Result<()> {
     let mut builder = LogBuilder::new();
     builder.format(format);
 
-    match matches
-              .value_of("log-level")
-              .or_else(|| global_matches.value_of("log-level")) {
+    match matches.value_of("log-level").or_else(|| {
```

Comment:
You seem to have a different version of rustfmt.  I plan to fix this all up ... when there aren't PRs outstanding to make life difficult for them.

Please undo the format changes.  It'll break the CI and make reviewing the code more difficult.